Examples/Algorithms/TrackFinding/src/TrackFindingAlgorithm.cpp

```diff
@@ -158,9 +169,27 @@ ActsExamples::ProcessCode ActsExamples::TrackFindingAlgorithm::execute(
 
     auto& tracksForSeed = result.value();
     for (auto& track : tracksForSeed) {
-      // Set the seed number, this number decrease by 1 since the seed number
-      // has already been updated
+      auto smoothingResult =
+          Acts::smoothTrack(ctx.geoContext, tracksTemp, track, logger());
+      if (!smoothingResult.ok()) {
+        ACTS_ERROR("Smoothing for seed "
+                   << iseed << " and track " << track.index()
+                   << " failed with error " << smoothingResult.error());
+        continue;
```

Comment:
Should this count towards `m_nFailedSeeds`? Similarly for `extrapolationResult` below?

Maybe only if all `tracksForSeed` fail, ie. none of them reach the track selector.

Previously, did it count as an error if smoothing or extrapolation failed for **any** track? That wasn't ideal, so progress!